lerna.json

```diff
@@ -6,5 +6,10 @@
     "integration/*",
     "demo/*"
   ],
+  "commands": {
+    "bootstrap": {
+      "hoist": "**"
```

Comment:
--hoist only works on dev dependencies so it shouldn't affect the regular ones.

If there's anything in devDependencies that's needed for the server to run then it should be a regular dependency.